src/index.html

```diff
@@ -4,11 +4,291 @@
+                  </svg>
+                </button>
+                <p class="process-card__more">Learn more</p>
+              </div>
+            </li>
+            <li class="process__item process-card">
+              <div class="process-card__number">003</div>
+              <ul class="process-card__list">
+                <li class="process-card__item">Marketing Strategy</li>
+                <li class="process-card__item">Email Marketing</li>
+                <li class="process-card__item">Paid Advertising</li>
+                <li class="process-card__item">Blog Content & SEO</li>
+              </ul>
+              <div class="process-card__extra">
+                <button class="process-card__button">
+                  <svg class="process__icon" width="16" height="16" viewBox="0 0 16 16" fill="white" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M8 0L6.59 1.41L12.17 7H0V9H12.17L6.59 14.59L8 16L16 8L8 0Z"/>
+                  </svg>
+                </button>
+                <p class="process-card__more">Learn more</p>
+              </div>
+            </li>
+            <li class="process__item process-card">
+              <div class="process-card__number">004</div>
+              <ul class="process-card__list">
+                <li class="process-card__item">Digital Communications</li>
+                <li class="process-card__item">Influencer Marketing</li>
+                <li class="process-card__item">Product Placements</li>
+                <li class="process-card__item">Strategic Partnerships</li>
+              </ul>
+              <div class="process-card__extra">
+                <button class="process-card__button">
+                  <svg class="process__icon" width="16" height="16" viewBox="0 0 16 16" fill="white" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M8 0L6.59 1.41L12.17 7H0V9H12.17L6.59 14.59L8 16L16 8L8 0Z"/>
+                  </svg>
+                </button>
+                <p class="process-card__more">Learn more</p>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="approach">
+        <div class="approach__container">
+          <div class="approach__text">
+            <h3 class="approach__subtitle">Approach</h3>
+            <h2 class="approach__title">We strive to create meaningful projects</h2>
+            <ul>
+              <li class="approach__item">
+                <div class="approach__icon approach__icon--shout"></div>
+                <p class="approach__description">
+                  We are sure that the brands built on a strong strategy, big ideas and
+                  simple human values always win. Only such brands change the game.
+                </p>
+              </li>
+              <li class="approach__item">
+                <div class="approach__icon approach__icon--calendar"></div>
```

Comment:
```suggestion
                <div class="approach__icon approach__icon-calendar"></div>
```
это просто доп название, а не состояние

Reply:
Ок, змінив назви